liveness.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+LAST_MODIFIED=$(date -r $LOG_FILE_PATH +%s)
+CURRENT=$(date +%s)
+
+# If the log file hasn't been modified in the last 2 minutes, liveness fails.
+if [ $(($LAST_MODIFIED + 120)) -ge $CURRENT ]
```

Comment:
are you sure you just can't equate this to a variable and echo it out, to provide the same results 
```suggestion
echo  $(($LAST_MODIFIED + 120)) -ge $CURRENT 
```
it might need a bit of bash formatting to work, but technically this is what this probably does anyway.

Reply:
For some reason the liveness probe wants a return value of 0 when the liveness check is successful and a return value of anything else when it's not so the logic would look kind of weird I think without the if.